server/src/test/java/org/opensearch/discovery/InitializeExtensionResponseTests.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.discovery;
+
+import org.opensearch.common.bytes.BytesReference;
+import org.opensearch.common.io.stream.BytesStreamInput;
+import org.opensearch.common.io.stream.BytesStreamOutput;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class InitializeExtensionResponseTests extends OpenSearchTestCase {
+    public void testInitialzeExtensionResponse() throws Exception {
```

Comment:
Minor nit: spelling error on initialize

Reply:
@varuntumbe I'm also fine fixing this in the upcoming PR that will add the tests

Reply:
Hi @ryanbogan ,

Sorry about that. For some reason, I am doing lot of awful spelling mistakes. I corrected this one. Hope everything else is intact !

Thanks and Regards
Varun.